src/lib.rs

```diff
@@ -148,7 +148,7 @@
 //!     }
 //!
 //!     // Define additional `ErrorKind` variants. The syntax here is
-//!     // the same as `quick_error!`, but the `from()` and `cause()`
+//!     // the same as `impl_error_chain_kind!`, but the `from()` and `cause()`
```

Comment:
Hum, I guess yes, if the doc is sufficiently detailed about the accepted syntax.